causalpy/pymc_models.py

```diff
@@ -117,3 +118,79 @@ def build_model(self, X, y, coords):
             sigma = pm.HalfNormal("sigma", 1)
             mu = pm.Deterministic("mu", pm.math.dot(X, beta), dims="obs_ind")
             pm.Normal("y_hat", mu, sigma, observed=y, dims="obs_ind")
+
+
+class InstrumentalVariableRegression(ModelBuilder):
+    """Custom PyMC model for instrumental linear regression"""
+
+    def build_model(self, X, Z, y, t, coords, priors):
+        """Specify model with treatment regression and focal regression data and priors
+
+        :param X: A pandas dataframe used to predict our outcome y
+        :param Z: A pandas dataframe used to predict our treatment variable t
+        :param y: An array of values representing our focal outcome y
+        :param t: An array of values representing the treatment t of
+                  which we're interested in estimating the causal impact
+        :param coords: A dictionary with the coordinate names for our
+                       instruments and covariates
+        :param priors: An optional dictionary of priors for the mus and
+                      sigmas of both regressions
+
+        e.g priors = {'mus': [[10, 0], [2, 0]], 'sigmas': [[1, 1], [1, 1]]
```

Comment:
Can we have the `priors = {...` as code formatted?

It looks like you have to use the `:code:` role (see here https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-code). Doing the following works for me:

    :code:`priors = {"mus": [0, 0], "sigmas": [1, 1], "eta": 2, "lkj_sd": 2}`

Reply:
Adjusted